### ISSUE TITLE
Currently the reward model validation loss for gpt2 plateaus early in training

### ISSUE THREAD

**Issue by scottviteri** [scottviteri/DemocraticConstitutionalAI#3]
Possible ways to fix:
1. Use a bigger model
2. Manually inspect reward model 
  i. Print differences in rewards directly (no -log(sigmoid(x)))
  ii. Look at the rewards on manually generated good and bad completions
3. Play with hyperparameters and babysit loss curve
